speedcord/http.py

```diff
@@ -47,6 +58,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 class HttpClient:
     def __init__(self, token, *, baseuri="https://discord.com/api/v7", loop=asyncio.get_event_loop()):
+        """
+        Created by speedcord.Client when started. Runs requests sent to it by creating Route 
+        objects and also handles rate limits with LockManager. 
+        :param token: The Discord Bot token.
+        :param baseuri: Discord's API uri. Generally should use the default. 
+        :param loop: an asyncio.AbstractEventLoop. get_event_loop() should be fine most of the time. 
```

Comment:
Change to something like `:param loop: the loop to use for callbacks` or something like that